source/ios/tools/AdaptiveCards.podspec

```diff
@@ -2,7 +2,7 @@
 Pod::Spec.new do |spec|
   spec.name             = 'AdaptiveCards'
 
-  spec.version          = '1.0.3'
+  spec.version          = '2.8.0-beta.3'
```

Comment:
so how do we update spec.version?

I don't see where it's being updated.

Reply:
This line should be updated manually ahead of release, following practice by Azure SDKs